app/gui/src/project-view/mock/mockSuggestions.json

```diff
@@ -1047,6 +1047,17 @@
     "documentation": "",
     "annotations": []
   },
+  {
+    "type": "method",
+    "module": "Standard.Table.Table",
+    "name": "input",
+    "arguments": [],
```

Comment:
I would put the argument here in case someone in the future would make a test with connecting to input args.